python/pfs/drp/stella/utils.py

```diff
@@ -65,3 +66,52 @@ def makeFiberTraceSet(pfsFiberTrace, maskedImage=None):
                 raise RuntimeError("FiberTrace %d: Failed to create trace from maskedImage")
         fts.addFiberTrace(ft)
     return fts
+
+def createPfsFiberTrace(dataId, fiberTraceSet, nRows):
+    pfsFiberTrace = pFT(dataId['calibDate'], dataId['spectrograph'], dataId['arm'])
+
+    ftf = fiberTraceSet.getFiberTrace(0).getFiberTraceFunction()
+    ftfc = ftf.fiberTraceFunctionControl
+
+    ftpfc = fiberTraceSet.getFiberTrace(0).getFiberTraceProfileFittingControl()
+
+    pfsFiberTrace.fwhm = 0.
+    pfsFiberTrace.threshold = 0.
+    pfsFiberTrace.nTerms = 0
+    pfsFiberTrace.saturationLevel = 0.
+    pfsFiberTrace.minLength = 0
+    pfsFiberTrace.maxLength = 0
+    pfsFiberTrace.nLost = 0
```

Comment:
If these are defaults, why aren't they in the `__init__` function?  You are using `pfs.datamodel.pfsFiberTrace.PfsFiberTrace` so why didn't you either add member variables to that class or inherit from it?